docs/IOS.md

```diff
@@ -0,0 +1,7 @@
+# iOS support
+
+## iOS setup: script for adjusting environment variables
+
+To integrate automatic setup of Hermes engine, a special env variable (`REACT_NATIVE_OVERRIDE_HERMES_DIR`) must be set to a proper path. We provide the script `react-native-node-api vendor-hermes --silent` which will output a single line, the path to Hermes directory.
+
+Each time you run XCode, make sure this is in place.
```

Comment:
This is not true, it only needs to be set when running "pod install" and we do that automatically - no need for manual steps on iOS, not even when running XCode.

```suggestion
```

```suggestion
```